workflows/nanoseq.nf

```diff
@@ -336,6 +349,9 @@ workflow NANOSEQ{
         ch_view_sortbam
             .map { it -> [ it[0], it[3] ] }
             .set { ch_sortbam }
+        ch_view_sortbam
+            .map { it -> [ it[0], it[3], it[4] ] }
+            .set { ch_nanopolish_sortbam }
```

Comment:
Are both of these needed or can it be made into an if statement?

Reply:
Yes, both of these are needed for the RNA track: `ch_sortbam` is for bigbed, bigwig, and RNA quantification, and RNA fusion detection while `ch_nanopolish_sortbam` is for RNA modification detection.